control-plane/agents/core/src/core/scheduling/mod.rs

```diff
@@ -45,8 +45,8 @@ pub(crate) trait ResourceFilter: Sized {
 /// Filter nodes used for replica creation
 pub(crate) struct NodeFilters {}
 impl NodeFilters {
-    /// Should only attempt to use online nodes
-    pub(crate) fn online(_request: &GetSuitablePoolsContext, item: &PoolItem) -> bool {
+    /// Should only attempt to use online nodes for pools
+    pub(crate) fn online_for_pool(_request: &GetSuitablePoolsContext, item: &PoolItem) -> bool {
```

Comment:
Sorry this is my fault, this should have been trait based, we'll address this later.

Reply:
Sure.